src/infrastructure/getNamespace.js

```diff
@@ -76,6 +79,40 @@ class sdkNamespace {
 
     return format.formatNamespace(namespaceInfo, namespaceNames)
   }
+
+  static getNamespacesFromIdWithLimit = async (limit, fromNamespaceId) => {
```

Comment:
same thing as mosaic.

it should able accept `namespaceId` or `namespace name`